src/components/utility/Button/Button.jsx

```diff
@@ -43,11 +53,12 @@ const Button = ({ elType, disabled = false, onClick, children, className }) => {
 };
 
 Button.propTypes = {
-  elType: PropTypes.oneOf(Object.keys(buttonColorStyle)).isRequired,
+  elType: PropTypes.oneOf(["white", "dark"]).isRequired,
```

Comment:
if it secondary it should not have an elType, I will work and fix the buttons

Reply:
since the elType  props is already required and I didn't want  to make changes that could affect the already added buttons but I can update the Buttons to reflect the same